Library/Styles/CheckoutStyles.swift

```diff
@@ -17,6 +17,21 @@ public enum CheckoutConstants {
       }
     }
   }
+
+  public enum PaymentSource {
+    public enum Card {
+      public static let width: CGFloat = 240
+    }
+
+    public enum ImageView {
+      public static let width: CGFloat = 64
```

Comment:
Little confused as to when we stopped using the grid for these, I know we have done this a fair amount... but why? 😄

Reply:
I think we only really use the grid to define positions and margins - when it comes to assets they have their own size so we use that 🤷‍♀